pkg/controllers/hypershiftdeployment_controller.go

```diff
@@ -87,6 +87,8 @@ func (r *HypershiftDeploymentReconciler) Reconcile(ctx context.Context, req ctrl
 		return ctrl.Result{}, nil
 	}
 
+	r.Log.Info("Reconciling")
```

Comment:
can we do `log.Info("Reconciling")`, just to be consistent with the rest.

Reply:
Changed in this pull request https://github.com/stolostron/hypershift-deployment-controller/pull/10